src/ripple/app/ledger/Ledger.cpp

```diff
@@ -261,13 +259,12 @@ Ledger::Ledger (
 }
 
 // Create a new ledger that follows this one
```

Comment:
Apropos of our conversation about terminology and vocabulary, this comment is slightly confusing in multiple ways. First, it probably belongs in the header and it should read: "create a successor ledger to the one provided".